metricbeat/module/system/_meta/config.yml

```diff
@@ -1,20 +1,15 @@
 - module: system
-  period: 10s
-  metricsets:
-    - cpu
-    - load
-    - memory
-    - network
-    - process
-    - process_summary
-    #- core
-    #- diskio
-    #- socket
-  processes: ['.*']
+  # processes: ['.*']
   process.include_top_n:
     by_cpu: 5      # include top 5 processes by CPU
     by_memory: 5   # include top 5 processes by memory
 
+#- module: system
```

Comment:
Not sure if the indentation is correct if the `#` is removed here. It should be 2 spaces before the `#`.

Reply:
I think identation is correct, if the `#` is removed it stays at the same level as other blocks.

Reply:
ah, you are right.